src/pages/Account/Profile.tsx

```diff
@@ -72,9 +72,13 @@ const Profile: React.FunctionComponent<PropsType> = ({ location }) => {
         <div className="login-welcome-subtitle">
           {`Swing back and edit your profile anytime you like.`}
         </div>
-        <button className="profile-picture-button summit-snap">
+        <a
+          className="profile-picture-button summit-snap"
+          href="https://virtual.githubphotobooth.com/virtual/capture/gr99n"
+          {...externalUrlAdditionalProps}
+        >
```

Comment:
should we open it in a new tab?

```suggestion
        <a
          className="profile-picture-button summit-snap"
          href="https://virtual.githubphotobooth.com/virtual/capture/gr99n"
          target="_blank"
          rel="noopener noreferer"
          {...externalUrlAdditionalProps}
        >
```

Reply:
@sunny-viktoryia thats what spreading in the external url props does

Reply:
aaa, ooops, my bad. 🙈 
thank you!!

Reply:
All good :)